### PR TITLE
EE4.3 Hstream

### PR DESCRIPTION
**Pull request by DDDHuang**

### CODE REVIEW

File: en_US/rule/backend_hstreamdb.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+HStreamDB is a streaming database designed for streaming data, with complete lifecycle management for accessing, storing, processing, and distributing large-scale real-time data streams. It uses standard SQL (and its stream extensions) as the primary interface language, with real-time as the main feature, and aims to simplify the operation and management of data streams and the development of real-time applications.
+
+![image](./assets/rule-engine/HStreamDB/HStream.png)
+
+For more details, please refer to [HStream official website](https://hstream.io/)。
+
+## Create HStreamDB Server
+
+[Deployment reference document](https://hstream.io/docs/en/latest/start/quickstart-with-docker.html). Can use docker local deployment, or cloud hosting deployment.
+Use the command to create hstream-client:
```
Comment (Time-Hu) [inline en_US/rule/backend_hstreamdb.md]: Refer to the following [documents](https://hstream.io/docs/en/latest/start/quickstart-with-docker.html) for deployment instructions, which provide tutorials on deploying with docker locally or on the cloud.

Use the command to start an hstream-client

File: en_US/rule/backend_hstreamdb.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+```bash
+docker run -it --rm --name some-hstream-cli --network host hstreamdb/hstream:v0.8.0 hstream-client --port 6570 --client-id 1
+```
+
+Access to the console:
```
Comment (Time-Hu) [inline en_US/rule/backend_hstreamdb.md]: When entering the console, help info like these will be shown:

File: en_US/rule/backend_hstreamdb.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+> CREATE STREAM demo_stream;
+demo_stream
+> SHOW STREAMS;
+demo_stream
+>
+```
+
+## Create HStreamDB resource
+
+Access EMQX Dashboard，Click on Rules Engine, Resources, Create, select HStreamDB Resources, enter the resource address and link pool.
```
Comment (Time-Hu) [inline en_US/rule/backend_hstreamdb.md]: first comma `，` -> `,`

Comment (Time-Hu) [reply en_US/rule/backend_hstreamdb.md]: click on `Rules Engine` > `Resources` > `Create`, then select resources type `HStreamDB` .

File: en_US/rule/backend_hstreamdb.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+SELECT
+
+  payload
+
+FROM
+
+  "#"
+```
+
+The SQL rules in the document are for example only, please write the SQL according to the business design.
```
Comment (Time-Hu) [inline en_US/rule/backend_hstreamdb.md]: for example only -> for demonstrations only

**Review by Time-Hu — CHANGES_REQUESTED** [review]

**Review by zhongwencool — APPROVED** [review]

**Pull request merged**